Bindings/simbody.i

```diff
@@ -32,22 +32,25 @@ namespace SimTK {
 //%include <Bindings/std.i>
 %include <SWIGSimTK/BigMatrix.h>
 %template(StdVectorVec3) std::vector<SimTK::Vec3>;
+
 namespace SimTK {
 %extend RowVectorBase<double> {
      double __getitem__(size_t i) {
          if(i >= $self->nelt())
-             throw SimTK::Exception::Base{"Index out of Range."};
+             throw std::out_of_range{"Index out of Range."};
 
          return $self->getElt(0, i);
      }
 
      void __setitem__(size_t i, double value) {
          if(i >= $self->nelt())
-             throw SimTK::Exception::Base{"Index out of Range."};
+             throw std::out_of_range{"Index out of Range."};
 
          $self->updElt(0, i) = value;
      }
  }
+%exception;
```

Comment:
This clears all previous exception-handling macros, right? Perhaps it would be helpful to add a comment that mentions that, since it is not obvious what this does.